packages/api-sdk/.devcontainer/README.md

```diff
@@ -0,0 +1,29 @@
+> **Remember to shutdown a GitHub Codespace when it is not in use!**
+
+# Dev Containers Quick Start
```

Comment:
This is the beginning of the generated SDK.  Maybe don't worry to much about a proper review here, but if curious take a look to see what the generated code looks like.